stat/stat.go

```diff
@@ -217,6 +217,59 @@ func Correlation(x, y, weights []float64) float64 {
 	return (sxy - xcompensation*ycompensation/sumWeights) / math.Sqrt(sxx*syy)
 }
 
+// KendallCorrelation returns the weighted Tau-a Kendall correlation between the
```

Comment:
Is the work "Correlation" needed? Can we just call it Kendall?

Reply:
Sure I'm fine with that.